tests/jsons/flex-duration-2/output.json

```diff
@@ -858,7 +858,16 @@
   "impossible": [
     {
       "day": "2022-10-01",
-      "outputs": []
+      "outputs": [
+        {
+          "taskid": 98,
+          "goalid": "3",
+          "title": "sleep",
+          "duration": 2,
+          "start": "2022-10-01T00:00:00",
+          "deadline": "2022-10-15T00:00:00"
+        }
+      ]
```

Comment:
@moaz-mokhtar It would be nice if this test case is fully possible (no impossible).  

A way to do that is by disregarding the calendar start/end in the goal's timeline and task_placer.  
So the sleep timeline would start on either:
- the goal.start if present - adjusted to the after time
- one day before calendar start - adjusted to the after time - if no goal.start is present

The calendar start/end would be taken into account in the output formatter - which would then 'chop off' the first day.

Reply:
We'll merge this PR and create a separate issue for this.